janus_core/calculations/phonons.py

```diff
@@ -223,15 +246,24 @@ def __init__(
             Default is False.
         minimize_kwargs : Optional[dict[str, Any]]
             Keyword arguments to pass to geometry optimizer. Default is {}.
+        n_qpoints : int
+            Number of q-points to sample along generated path, including end points.
+            Unused if `paths` is specified. Default is 51.
+        paths : PathLike | None
```

Comment:
Maybe `qpoint_path_file` or similar? `paths` has a few too many other meanings.

Reply:
Good point, that's probably sensible

Reply:
Do you think `qpoint_file` (`--qpoint-file`) is clear enough? Mostly to avoid long CLI options